03_ScienceExamples/DESI/01_Intro_to_DESI_EDR.ipynb

```diff
@@ -1638,9 +1638,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "DESI 23.1",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
Re-run this NB on the DL NB server, so that it can use the DESI 23.x kernel (this is important, because specific kernels usually have specific S/W installed)/